File: co.edu.uniandes.fixml/test/co/edu/uniandes/fixml/src-gen/test4/CPlus/Rule80A.cpp
#include <iostream> 

using namespace std;

class Rule80A {

	private:
  	string Value_;
  	string FIXTag_;
  	string DataType_;
 	
 	public: 

  	// default constructor
	Rule80A () {
		Value_ = "A";
		FIXTag_ = "47";
		DataType_ = "char";
  	
  	}
  
  	// by parameters
  	Rule80A (		string Value_p, string FIXTag_p, string DataType_p){
	  
		Value_ = Value_p;
		FIXTag_ = FIXTag_p;
		DataType_ = DataType_p;
	  
  	}
};

File: co.edu.uniandes.fixml/test/co/edu/uniandes/fixml/src-gen/test4/Java/Rule80A.java
package object;



// Class for tag Rule80A
public class Rule80A {

  String Value;
  String FIXTag;
  String DataType;


  // default constructor
  public Rule80A ( String Value, String FIXTag, String DataType ) {
    this.Value = Value;
    this.FIXTag = FIXTag;
    this.DataType = DataType;
  
  }


  public Rule80A ( ) {   
  
this.Value = "A";
this.FIXTag = "47";
this.DataType = "char";
  
  }
      

} 